app/controllers/ReadOnly.scala

```diff
@@ -23,13 +24,50 @@ object ReadOnlyApi extends Controller {
   }
 
   def mergesAsXml(since: Long) = Action {
-    val merges = JobRepository.getMerges().map { job =>
+    val merges = JobRepository.getMerges.map { job =>
       Merge(job)
     }.filter(_.started.getMillis > since)
 
-    Ok(<merges>
-      {merges.map( x => x.asExportedXml)}
+    Ok(
+      <merges>
+        {merges.map( x => x.asExportedXml)}
       </merges>
     )
   }
+
+  def deletesAsXml(since: Long) = Action {
+    val deletes = JobRepository.getDeletes.map { job =>
```

Comment:
This isn't a record of all Jobs that have happened in the past, they are deleted once complete (the only reason they're there now is due to other issues with merge/delete)

Tag Audit is a better source of this information